src/utilities/fileSystem.utils.ts

```diff
@@ -33,6 +34,28 @@ export const createDirectory = async (outputPath: string): Promise<void> => {
 	}
 };
 
+export const openFileExplorer = (directoryPath: string): void => {
+	const normalizedPath = path.normalize(directoryPath);
+	const quotedPath = `"${normalizedPath}"`;
+	let command;
+
+	switch (process.platform) {
```

Comment:
@ig-ra Please lmk wdyt about the implementation, and do we need better treatment in case we run it on a Linux station?

Reply:
Discussed in slack - conclusion:
Xdg-open is the common one - No need to support others